ouroboros-consensus/test-consensus/Test/ThreadNet/General.hs

```diff
@@ -169,7 +169,11 @@ data TestConfigBlock blk = TestConfigBlock
   }
 
 data Rekeying blk = forall opKey. Rekeying
-  { rekeyUpd ::
+  { rekeyOracle
```

Comment:
TODO Can we simplify away this callback? Specifically, can we create the cert as part of the test-configuration (eg make `NodeRestarts` a container, with items as a new argument to the `NodeRekey` ctor)?